toolkit/scripts/toolchain/cross/busybox.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/networking/tls_aesgcm.c b/networking/tls_aesgcm.c
+index 5ddcdd2..c8030e7 100644
+--- a/networking/tls_aesgcm.c
++++ b/networking/tls_aesgcm.c
+@@ -10,6 +10,7 @@ typedef uint8_t byte;
+ typedef uint32_t word32;
+ #define XMEMSET memset
+ #define XMEMCPY memcpy
++#define LONG_BIT CHAR_BIT * sizeof(long)
```

Comment:
Is this define needed when we're not cross compiling (i.e. using an aarch64 hosted toolset)?

I'm asking because I'm wondering if LONG_BIT is supposed to be defined in a toolchain header. I looked online and it wasn't immediately clear to me. I want to make sure the include paths are correct so the right headers can be found.